src/transpilation/utils.ts

```diff
@@ -19,6 +19,7 @@ interface BaseFile {
     code: string;
     sourceMap?: string;
     sourceFiles?: ts.SourceFile[];
+    isRawFile?: boolean;
```

Comment:
why is this even required?

Reply:
This is for the json file. Before `getEmitPath` expects absolute paths and turns filenames into lua, and the json file name is not absolute. This allows treating the json file differently. 
If you can think of a better way to handle this let me know.